app/scripts/services/journalUploadService.ts

```diff
@@ -0,0 +1,150 @@
+      });
+  }
+
+  // Load all account transfer reports from the server
+  loadAllAccountTransferReports(
+    conferenceId: string,
+  ): IPromise<Array<JournalReport>> {
+    this.$rootScope.loadingMsg = 'Loading Reports';
+
+    return this.$http
+      .get<Array<JournalReport>>(
+        `conferences/${conferenceId}/account/transfer/reports`,
+      )
+      .then((response) => response.data)
+      .finally(() => {
+        this.$rootScope.loadingMsg = '';
+      });
+  }
+
+  // Load a single account transfer report from the server given its id
+  loadAccountTransferReport(
+    conferenceId: string,
+    reportId: string,
+  ): IPromise<JournalReport> {
+    return this.loadAccountTransferReportFromUrl(
+      `conferences/${conferenceId}/promotion/report/${reportId}`,
+    );
+  }
+
+  // Load a single account transfer report from the server given its URL
+  loadAccountTransferReportFromUrl(reportUrl: string): IPromise<JournalReport> {
+    this.$rootScope.loadingMsg = 'Loading Report';
+
+    return this.$http
+      .get<JournalReport>(reportUrl)
+      .then((response) => response.data)
+      .finally(() => {
+        this.$rootScope.loadingMsg = '';
+      });
+  }
+
+  // Returned promise resolves to null if the submission is still processing on the server, but the request timed out
+  submitAccountTransfers(
+    accountTransfers: Array<AccountTransfer>,
+  ): IPromise<JournalReport | null> {
+    this.$rootScope.loadingMsg = 'Submitting account transfers';
+    // Filter out included key from account transfer object
+    const filteredAccountTransfers = accountTransfers.map((accountTransfer) =>
+      omit(accountTransfer, ['included']),
+    );
+
+    return this.$http
+      .post<string>('account/transfers', filteredAccountTransfers)
+      .then((response) => this.loadAccountTransferReportFromUrl(response.data))
+      .catch((errorResponse) => {
+        const error = errorResponse.data?.error;
+
+        this.$rootScope.loadingMsg = '';
+        this.modalMessage.error({
+          title: error
+            ? 'Error Submitting Account Transfers'
+            : 'Journal Upload Delay',
+          message:
+            error ??
+            '<p>Journal Upload process time varies by the size of the list submitted and your submission is taking longer than expected to process. The system is still working to complete your request.</p>' +
+              '<br>' +
+              '<p>The transactions submitted in this Journal will still be listed on the "New Report" transactions list until this submission is complete. Please DO NOT submit the same transactions again while waiting or charges will be duplicated.</p>' +
+              '<br>' +
+              '<p>Check back in a few minutes to see if your submitted report is in the dropdown box beside "Report Creation Date." Once your submitted report shows up in the dropdown box all completed transactions will no longer be listed in the "New Report" transactions list.</p>',
```

Comment:
```suggestion `<p>Journal Upload process time varies by the size of the list submitted and your submission is taking longer than expected to process. The system is still working to complete your request.</p>
<br>
<p>The transactions submitted in this Journal will still be listed on the "New Report" transactions list until this submission is complete. Please DO NOT submit the same transactions again while waiting or charges will be duplicated.</p>
<br>
<p>Check back in a few minutes to see if your submitted report is in the dropdown box beside "Report Creation Date." Once your submitted report shows up in the dropdown box all completed transactions will no longer be listed in the "New Report" transactions list.</p>`
```

You could do this instead of having seperate string with additions symbols.